apis/core/lib/domain/dimension-mapping/update.ts

```diff
@@ -29,7 +29,7 @@ export async function update({
     throw new error.BadRequest('Unexpected value of schema:about')
   }
 
-  if (!managedDimension.equals(dimensionMappings.managedDimension)) {
+  if (!managedDimension || !managedDimension.equals(dimensionMappings.managedDimension)) {
```

Comment:
Worthy of adding a unit test?

Reply:
Probably yes. I tried and then got lazy. Is there a reason why the "when" of these unit tests is in the `beforeEach`? That seems really wrong and makes it hard to add new test cases in the same `describe`.

Reply:
Yea, that doesn't make sense. Especially that there is only one test case in that `describe`.

Definitely move the `await update()` inside the `it()`